apps/governance-e2e/src/integration/view/home.cy.ts

```diff
@@ -161,7 +161,7 @@ context('Home Page - verify elements on page', { tags: '@smoke' }, function () {
       cy.getByTestId('menu-drawer').should('be.visible');
     });
 
-    it('should have link for proposal page', function () {
+    it.skip('should have link for proposal page', function () {
```

Comment:
Add some comments about why these are skipped, with links to the relevant issue?

Reply:
Thanks Matt. Addressed both your comments in this ticket: https://github.com/vegaprotocol/frontend-monorepo/issues/3660